app/dawn-hub/Views/OnboardingViews/Step3_ChooseDefaultSearchView/ChooseDefaultSearchView.jsx

```diff
@@ -18,13 +18,30 @@ import RadioButton from '../../../../shared-components/RadioButton';
 import { ONBOARDING, CHOOSE_PLAN } from '../../OnboardingView/OnboardingConstants';
 import {
 	SEARCH_GHOSTERY,
+	SEARCH_BING,
 	SEARCH_YAHOO,
 	SEARCH_STARTPAGE,
-	SEARCH_BING,
 	SEARCH_OTHER
 } from './ChooseDefaultSearchConstants';
 import { Modal } from '../../../../shared-components';
 
+const mapSearchToSetupNumber = {
+	Ghostery: 1,
+	Bing: 2,
+	Yahoo: 3,
+	Startpage: 4,
+	'DuckDuck Go': 6,
+	Ecosia: 7,
+	Ekoru: 8,
+	Gibiru: 9,
+	Google: 10,
+	OneSearch: 11,
+	Privado: 12,
+	Qwant: 13,
+	'Search Encrypt': 14,
+	Tailcat: 15,
+};
+
```

Comment:
For the "other" dropdown, we use the `name` field value in the array returned by `browser.search.get()`. I think some of those names might be slightly different. For example, I think the DDG `name` is `DuckDuckGo`. Can you please double check Startpage, DDG, and Search Encrypt in particular? I believe you can install extensions to add engines to the default search selection list if they are not there by Firefox default

Reply:
Also, why not use the constants you defined?

Reply:
Double checked all of them 👍 `Search Encrypt` doesn't seem to be available on the Firefox or dawn store right now, so I left it as a TODO. I didn't want to use the constants initially because the name's wouldn't be prettified (i.e. `gibiru.com`), but wound up using them anyway for readability

Reply:
Thank you!!